static/sass/custom/_patterns_icons.scss

```diff
@@ -1,52 +0,0 @@
-// XXX Ant: 26.06.2019 - Need more control over icons in the footer
```

Comment:
This issue is still open. Do we not need this control any more?

Reply:
This whole stylesheet was for custom styling of custom grayscale social icons - coloured social icons have since been added to Vanilla so we now just use those directly.

Reply:
I'm +100 on using Vanilla wholesale but originally they wanted the social icons to be grey and on hover to style the brand color but let's leave it.